web/directory/management/commands/parse_coop_csv.py

```diff
@@ -139,14 +167,15 @@ def get_address_pks(file_path, city_pks):
                 # If there are no lat or lon coords provided, attempt to figure
                 # them out
                 if not lat or not lon:
-                    svc = LocationService()
-                    ret = svc.get_coords(
-                        address=street,
-                        city=city,
-                        zip=postal_code,
-                        state_code=state_id,
-                        country_code="US"
-                    )
+                    #svc = LocationService()
+                    #ret = svc.get_coords(
+                    #    address=street,
+                    #    city=city,
+                    #    zip=postal_code,
+                    #    state_code=state_id,
+                    #    country_code="US"
+                    #)
+                    ret=osGeoLoc(street+", "+city+", "+state_id+" "+postal_code)
```

Comment:
Most excellent!  One small thing since I'd like to reuse this code in the maps application as well -- could you uncomment lines 171 through 177 and place your code in the "get_coords" method of the LocationService?  That way both the script and the application can take advantage of what you have.  Thanks, - Dave